02_activities/assignments/assignment.sh

```diff
@@ -1,24 +1,35 @@
 #!/bin/bash
-set -x
-
+ set -x
+m
 ################
 # Instructions #
 ################
 # > Edit this file below to complete the homework assignment
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
-
+ mkdir dir1, mkdir dir2, mkdir dir3, mkdir dir4, mkdir dir5
 # 2. List the contents of the parent directory to verify the presence of the 5 directories
-
+ls
 # 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
+$ touch dir2/file1.txt
+$ touch dir2/file2.txt
+$ touch dir2/file3.txt
+$ touch dir2/file4.txt
+$ touch dir2/file5.txt
```

Comment:
Please don't resolve my comment before you have made changes to your code. Per my previous comment, remove the $ from your code and I will mark as complete.

Also, Q1 does not create five directories name dir1 ... dir5. You are creating these files with a comma at the end and a directory named mkdir

Your Q6 should specify that the file you want to remove is in dir2.

Reply:
I have made corrections. Can you review please. Thank you